source/ui/SGM/SGMXASScanConfigurationView.cpp

```diff
@@ -11,6 +13,15 @@ SGMXASScanConfigurationView::SGMXASScanConfigurationView(SGMXASScanConfiguration
 	configuration_ = configuration;
 	detectors_ = 0;
 
+	gratingAngleControl_ = new SGMGratingAngleControl(this);
```

Comment:
I'm not super happy about this myself.
The are used elsewhere, but only in the energy coordinator application. Generally I kept all the calculations in there and hoped that the acquaman side could just dumbly throw values at the exposed energy PVs. Unfortunately, in order to do this calculation in the view the grating translation, grating angle encoder counts per step, and the grating acceleration are all required. 

I would like to refactor this in the future, but within the scope of this issue, this was the best arrangement.